### PR TITLE
add components to welcome page 

### PR DESCRIPTION
**Pull request by leeannachen**
breaking a page down into smaller pull requests

### CODE REVIEW

**Comment by leeannachen** [comment]
@googlebot I signed it!

File: proj/src/main/webapp/static/images/friendsChatting.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0938-7.726563 7.726563-7.726563h67.214843c4.265626 0 7.726563 3.457032 7.726563 7.726563 0 4.265625-3.460937 7.726563-7.726563 7.726563zm0 0"/></g><path d="m305.15625 426.234375v-77.179687c0-27.570313 22.351562-49.917969 49.921875-49.917969h102.894531c27.570313 0 49.921875 22.347656 49.921875 49.917969v70.527343c0 27.570313-22.351562 49.921875-49.921875 49.921875h-176.726562zm0 0" fill="#f2c42c"/><path d="m460.414062 356.785156h-107.777343c-4.269531 0-7.726563-3.460937-7.726563-7.726562 0-4.269532 3.457032-7.726563 7.726563-7.726563h107.777343c4.269532 0 7.726563 3.457031 7.726563 7.726563 0 4.265625-3.460937 7.726562-7.726563 7.726562zm0 0" fill="#efedee"/><path d="m460.414062 392.046875h-107.777343c-4.269531 0-7.726563-3.460937-7.726563-7.726563 0-4.265624 3.457032-7.726562 7.726563-7.726562h107.777343c4.269532 0 7.726563 3.460938 7.726563 7.726562 0 4.265626-3.460937 7.726563-7.726563 7.726563zm0 0" fill="#efedee"/><path d="m460.414062 427.3125h-67.214843c-4.265625 0-7.726563-3.460938-7.726563-7.726562 0-4.269532 3.460938-7.726563 7.726563-7.726563h67.214843c4.265626 0 7.726563 3.457031 7.726563 7.726563 0 4.265624-3.460937 7.726562-7.726563 7.726562zm0 0" fill="#efedee"/></svg>
```
Comment (showermat) [inline proj/src/main/webapp/static/images/friendsChatting.svg:1]: What's the source of this image?  Did you draw it?  If not, there needs to be attribution and you need to make sure that it's license-compatible before committing.

File: proj/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,3 +1,65 @@
+
+html{
+  font-family: 'Montserrat', sans-serif;
+}
+
+.screen{
+  background: grey;
```
Comment (showermat) [inline proj/src/main/webapp/style.css:7]: How British of you.

**Review by showermat — COMMENTED** [review]
Looks great, but I need an answer for the image sourcing.

File: proj/src/main/webapp/static/images/friendsChatting.svg (hunk 1 above)
Comment (leeannachen) [reply proj/src/main/webapp/static/images/friendsChatting.svg:1]: Thank you for reminding me.  The website has free licensing with attribution.  I am going to add the tag in the footer.

Comment (showermat) [reply proj/src/main/webapp/static/images/friendsChatting.svg:1]: That looks good to me!

**Review by showermat — APPROVED** [review]

**Pull request merged**